Source/Core/DolphinQt/Config/Graphics/HacksWidget.cpp

```diff
@@ -106,11 +106,14 @@ void HacksWidget::CreateWidgets()
   m_vertex_rounding = new GraphicsBool(tr("Vertex Rounding"), Config::GFX_HACK_VERTEX_ROUDING);
   m_save_texture_cache_state =
       new GraphicsBool(tr("Save Texture Cache to State"), Config::GFX_SAVE_TEXTURE_CACHE_TO_STATE);
+  m_native_texture_sampling =
```

Comment:
Since we already use "Native" to mean the emulated system (ex: "Native Resolution"), I wonder if this should be called something else

Reply:
Maybe "Fast texture sampling" (with the alternative still being "Manual texture sampling") would work; it's not _necessarily_ true (since sometimes manual texture sampling is faster) but it'd make things more obvious to end-users who assume that fast means less accurate.

Reply:
Thanks, I just saw this.  I can't speak for anyone else but I like this a lot better!